modules/custom/openy_addthis/openy_addthis.install

```diff
@@ -10,7 +10,7 @@
  */
 function openy_addthis_install() {
   // Add our base field to the schema.
-  \Drupal::service('entity.definition_update_manager')->applyUpdates();
+  // TODO: \Drupal::service('entity.definition_update_manager')->applyUpdates();
```

Comment:
I think there should be code according to https://www.drupal.org/node/3034742

> Whenever an entity type or field storage definition needs to be created, changed or deleted, it has to be done with an explicit update function

Reply:
And we have no uncontrolled updates for storage definitions @hamrant 
So we can remove this code